systemtest/src/main/resources/olm/subscription.yaml

```diff
@@ -20,4 +20,5 @@ spec:
         value: "${STRIMZI_FULL_RECONCILIATION_INTERVAL_MS}"
       - name: STRIMZI_OPERATION_TIMEOUT_MS
         value: "${STRIMZI_OPERATION_TIMEOUT_MS}"
-
+      - name: STRIMZI_RBAC_SCOPE
+        value: "${STRIMZI_RBAC_SCOPE}"
```

Comment:
I think this part is confusing. The OLM deployment does not allow you to change the settings of the ClusterRoles / ClusterRoleBindings. It always deploys the operator in cluster-scope. So I'm not sure any tests with it run with the operator configured for the namespaced-scope make sense. This might confuse someone into thinking they do. So I would probably remove this.

Reply:
Well technically a user could modify their OLM installation like this, and its a convenient way to test this feature.
Should we only test this via the 'install' folder method?

Reply:
Well, OLM doesn't support this. So you would need to have a completely separate operator bundle in OLM. Would you really run the system tests against that? Do they even support it in some reasonably easy way?